lib/_http_server.js

```diff
@@ -223,6 +231,13 @@ ServerResponse.prototype._finish = function _finish() {
     });
   }
   OutgoingMessage.prototype._finish.call(this);
+  if (isTraceHTTPCategory() && this._traceEventId) {
```

Comment:
Should we replace this check with `typeof this._traceEventId !== 'undefined'`?

Reply:
Thanks ! Done !